jsonschema-core.xml

```diff
@@ -1789,7 +1789,9 @@
                     <section title="References to Possible Non-Schemas">
                         <t>
                             Subschema objects (or booleans) are recognized by their use with known
-                            applicator keywords.  These keywords may be the standard applicators
+                            applicator keywords, or with placeholder keywords such as
```

Comment:
Sorry for my late feedback. To be honest I am not sure if this would clarify things considerably compared to before. I weren't able to draw the line in the opposite direction (as a reader of section *Schema Re-Use With "$defs"* right to this section and the essence of its meaning). But that might be caused by my personal situation of "not being in the specs" in such depth, so isn't meant to disqualify your answer. Though, the newly introduced term *placeholder keyword*  would require its own definition.

I would accept this PR as closing my issue but would also accept the issue being closed without any further action if majority doesn't see any necessity for clarification.

Thank you.

Just a little note: I can't comment on things in a timely manner ATM so don't make your progress depend on me.

Reply:
Thanks, @about-code!  That's helpful feedback, I'll tweak it a bit more (we don't want people to have to already be experts when reading the standard- it's not a tutorial but it should be accessible).  If you have time to look at the update, great, but thanks for letting me know that you might not be available.